### PR TITLE
Update choosing-msal-dotnet.md

### PR DESCRIPTION
**Pull request by gladjohn**
Update choosing-msal-dotnet.md

### CODE REVIEW

File: msal-dotnet-articles/getting-started/choosing-msal-dotnet.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Just take the code you need from Microsoft Identity Web to update your app. Here
...
 
-You're building and SDK for confidential client applications and want to use MSAL.NET low level APIs. In MSAL.NET, an in-memory token cache is provided by default, however, in the case of web apps or web APIs, caching should be handled differently than for public client applications (desktop or mobile apps) as it requires to be partitioned correctly. It's highly recommended to leverage a token cache serializer, which can be a distributed cache, (e.g. Redis, Cosmos, or SQL Server, distributed in memory cache), or a correctly partitioned in memory cache.
+You are building a SDK for confidential client applications and want to use MSAL.NET low level APIs. In MSAL.NET, an in-memory token cache is provided by default, however, in the case of web apps or web APIs, caching should be managed differently than for public client applications (desktop or mobile apps) as it requires to be partitioned correctly. It is highly recommended to leverage a token cache serializer, which can be a distributed cache, (e.g., Redis, Cosmos, or SQL Server, distributed in memory cache), or a correctly partitioned in memory cache.
```
Comment (jmprieur) [inline msal-dotnet-articles/getting-started/choosing-msal-dotnet.md:98]: the contracted forms (You're, It's) are recommended in Microsoft Learn

File: msal-dotnet-articles/getting-started/choosing-msal-dotnet.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ You're building a confidential client application (Web app, web API, daemon/serv
   - generates WWW-authenticate headers in APIs (CA, CAE)
   - Protect gRPC services and Azure functions
 - Web app/API calling downstream APIs (including graph except for B2C)
-  - Call downstream APIs without having to handle authentication/tokens yourself.
+  - Call downstream APIs without having to manage authentication/tokens yourself.
   - Integrates with the graph SDK, and the Azure SDKs
   - Describe the client credentials, and Microsoft.Identity.Web fetches them for you (for
-    instance certificates from KeyVault, or workload identity federation with AKS and MSI)
+    instance certificates from Key Vault, or workload identity federation with [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/products/kubernetes-service) and [Managed Identities](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview))
```
Comment (jmprieur) [inline msal-dotnet-articles/getting-started/choosing-msal-dotnet.md:57]:
```suggestion
    instance certificates from Key Vault, or [workload identity federation](https://learn.microsoft.com/azure/active-directory/workload-identities/workload-identity-federation) with [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/products/kubernetes-service) and [Managed Identities](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview))
```

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @gladjohn 
I've added a suggestion

**Review by pmaytak — APPROVED** [review]

File: msal-dotnet-articles/getting-started/choosing-msal-dotnet.md [hunk 3]
```diff
@@ -51,10 +51,10 @@ You're building a confidential client application (Web app, web API, daemon/serv
   - generates WWW-authenticate headers in APIs (CA, CAE)
   - Protect gRPC services and Azure functions
 - Web app/API calling downstream APIs (including graph except for B2C)
-  - Call downstream APIs without having to handle authentication/tokens yourself.
+  - Call downstream APIs without having to manage authentication and tokens yourself.
   - Integrates with the graph SDK, and the Azure SDKs
   - Describe the client credentials, and Microsoft.Identity.Web fetches them for you (for
-    instance certificates from KeyVault, or workload identity federation with AKS and MSI)
+    instance certificates from Key Vault, or [workload identity federation](https://learn.microsoft.com/azure/active-directory/workload-identities/workload-identity-federation) with [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/products/kubernetes-service) and [Managed Identities](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview))
```
Comment (localden) [inline msal-dotnet-articles/getting-started/choosing-msal-dotnet.md:57]:
```suggestion
instance certificates from Key Vault, or [workload identity federation](/azure/active-directory/workload-identities/workload-identity-federation) with [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/products/kubernetes-service) and [Managed Identities](/azure/active-directory/managed-identities-azure-resources/overview))
```

**Review by localden — CHANGES_REQUESTED** [review]

**Comment by gladjohn** [comment]
@localden I am not seeing the change you requested for

**Pull request merged**